homeassistant/components/nederlandse_spoorwegen/sensor.py

```diff
@@ -40,11 +45,16 @@
 ROUTES_SCHEMA = vol.All(cv.ensure_list, [ROUTE_SCHEMA])
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend(
-    {vol.Required(CONF_API_KEY): cv.string, vol.Optional(CONF_ROUTES): ROUTES_SCHEMA}
+    {vol.Required(CONF_API_KEY): cv.string, vol.Required(CONF_ROUTES): ROUTES_SCHEMA}
```

Comment:
I think this was a bug. What is the point of the integration if `CONF_ROUTES` is `None`?

Reply:
🤷 It is also documented as optional. I suggest to remove this change from this PR

or fix it as 

```py
for departure in config.get(CONF_ROUTES, {}):
```

Reply:
I've adjusted the PR - but maybe @YarmoM would consider a follow-up PR to make it compulsory (or explain why it is optional)